backend/match4everyone/settings/common.py

```diff
@@ -98,6 +103,7 @@
         },
     },
 ]
+print(TEMPLATES)
```

Comment:
Should that be here? :)

Reply:
You probably had an old version in your cache, this was already removed in 58fad7d :+1: